src/fenicsxconcrete/sensor_definition/stress_sensor.py

```diff
@@ -34,17 +34,37 @@ def measure(self, problem: MaterialProblem) -> None:
             t : time of measurement for time dependent problems, default is 1
         """
         # project stress onto visualization space
+        stress, mandel_stress = None, None
         try:
             stress = problem.q_fields.stress
             assert stress is not None
```

Comment:
The q_fields are defined in the dataclass https://github.com/BAMresearch/FenicsXConcrete/blob/a41414a4addb4c84dec5975a8b63d100675e4238/src/fenicsxconcrete/finite_element_problem/base_material.py#L42 and therefore don't need to be in a try-block
```suggestion
stress = problem.q_fields.stress
```